src/components/TokenAdministrative.js

```diff
@@ -98,15 +99,37 @@ class TokenAdministrative extends React.Component {
    * Update token state after didmount or props update
    */
   updateWalletInfo = () => {
-    const mintUtxos = this.props.wallet.getMintAuthority(this.props.token.uid, { many: true });
-    const mintCount = mintUtxos ? mintUtxos.length : 0;
+    if (!this.props.token && !this.props.token.uid) {
+      return;
+    }
+
+    const { uid } = this.props.token;
+
+    let mintCount = 0;
+    let meltCount = 0;
+
+    const tokenBalance = this.props.tokensBalance[uid];
 
-    const meltUtxos = this.props.wallet.getMeltAuthority(this.props.token.uid, { many: true });
-    const meltCount = meltUtxos ? meltUtxos.length : 0;
+    if (this.props.useWalletService) {
+      // Wallet Service
+      // TODO: This should come from the wallet service and display the count properly
+      const canMintUtxos = tokenBalance.mint;
+      const canMeltUtxos = tokenBalance.melt;
+
+      mintCount = canMintUtxos ? 1 : 0;
+      meltCount = canMeltUtxos ? 1 : 0;
+    } else {
+      // Old Facade
+      const mintUtxos = this.props.wallet.getMintAuthority(this.props.token.uid, { many: true });
+      const meltUtxos = this.props.wallet.getMeltAuthority(this.props.token.uid, { many: true });
+
+      mintCount = mintUtxos ? mintUtxos.length : 0;
+      meltCount = meltUtxos ? meltUtxos.length : 0;
+    }
 
-    const tokenBalance = this.props.token.uid in this.props.tokensBalance ? this.props.tokensBalance[this.props.token.uid].available : 0;
+    const balance = this.props.token.uid in this.props.tokensBalance ? this.props.tokensBalance[this.props.token.uid].available : 0;
 
-    this.setState({ mintCount, meltCount, balance: tokenBalance });
+    this.setState({ mintCount, meltCount, balance: balance });
```

Comment:
This was refactored on this PR: https://github.com/HathorNetwork/hathor-wallet/pull/298 to have the same signatures on both facades